cmd/sops/main.go

```diff
@@ -78,6 +79,10 @@ func main() {
    (you need to setup google application default credentials. See
     https://developers.google.com/identity/protocols/application-default-credentials)
 
+   To encrypt or decrypt a document with Azure Key Vault, specify the
+   Azure Key Vault key URL in the --azure-kv flag or in the SOPS_AZURE_KEYVAULT_URL
+   environment variable.
```

Comment:
Noticed now that I'm missing the environment variables used to authenticate. Will add that.